src/runtime_src/tools/xclbinutil/SectionVenderMetadata.cxx

```diff
@@ -273,7 +264,6 @@ SectionVenderMetadata::writeMetadata(std::ostream& _oStream) const
   }
 
   vender_metadata* pHdr = reinterpret_cast<vender_metadata*>(m_pBuffer);
```

Comment:
auto

Reply:
Addressed this issue and others like it in the code base.